packages/x-components/src/design-system/utilities/box-shadow.scss

```diff
@@ -4,7 +4,7 @@
   box-shadow: none !important;
 }
 .x-shadow {
-  @for $i from 1 through 10 {
+  @for $i from 0 through 10 {
```

Comment:
Why did you add a `shadow-0`? you have a `shadow-none` just above

Reply:
Didn't realize, changed